### PR TITLE
Updated assisted-installer default image tag from stable to latest

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

File: internal/host/hwinfocmd.go [hunk 1]
```diff
@@ -21,6 +21,6 @@ func (h *hwInfoCmd) GetStep(ctx context.Context, host *models.Host) (*models.Ste
 	step := &models.Step{}
 	step.StepType = models.StepTypeHardwareInfo
 	step.Command = "podman"
-	step.Args = strings.Split("run,--rm,--privileged,--quiet,--net=host,-v,/var/log:/var/log,quay.io/ocpmetal/hardware_info:stable,/usr/bin/hardware_info", ",")
+	step.Args = strings.Split("run,--rm,--privileged,--quiet,--net=host,-v,/var/log:/var/log,quay.io/ocpmetal/hardware_info:latest,/usr/bin/hardware_info", ",")
```
Comment (filanov) [inline internal/host/hwinfocmd.go:24]: @ori-amizur is it part of you pr as well?

**Comment by filanov** [comment]
@ori-amizur @eranco74 you both touch the same area, please sync.

File: internal/host/hwinfocmd.go (hunk 1 above)
Comment (ori-amizur) [reply internal/host/hwinfocmd.go:24]: yes

Comment (ori-amizur) [reply internal/host/hwinfocmd.go:24]: yes

Comment (ronniel1) [reply internal/host/hwinfocmd.go:24]: @eranco74 PR should be merged first

Comment (ronniel1) [reply internal/host/hwinfocmd.go:24]: @eranco74 shouldn't it take the tag from the environment variable?

Comment (eranco74) [reply internal/host/hwinfocmd.go:24]: It should get it from env.
I'll update

File: internal/host/hwinfocmd.go [hunk 2]
```diff
@@ -2,25 +2,30 @@ package host
...
 )
 
-type hwInfoCmd baseCmd
+type hwInfoCmd struct {
+	baseCmd
+	instructionConfig InstructionConfig
+}
 
-func NewHwInfoCmd(log logrus.FieldLogger) *hwInfoCmd {
+func NewHwInfoCmd(log logrus.FieldLogger, instructionConfig InstructionConfig) *hwInfoCmd {
```
Comment (filanov) [inline internal/host/hwinfocmd.go:18]: why does HW info need `InstructionConfig`? why not just set the image?

Comment (eranco74) [reply internal/host/hwinfocmd.go:18]: It should get the image from env and I wanted to have all the instructions env overrides in one place.
Think I should move it the hwinfocmd.go?

Comment (filanov) [reply internal/host/hwinfocmd.go:18]: no, i think you can have a single struct like `InstructionConfig` but specific command should get only the relevant fields no?

Comment (eranco74) [reply internal/host/hwinfocmd.go:18]: Agree, updating

File: internal/host/instructionmanager.go [hunk 3]
```diff
@@ -28,15 +28,16 @@ type InstructionManager struct {
 	stateToSteps stateToStepsMap
 }
 type InstructionConfig struct {
```
Comment (filanov) [inline internal/host/instructionmanager.go:30]: @ori-amizur is `quay.io/ocpmetal/inventory` should be used somewhere?

**Pull request merged**